notebooks/official/ml_metadata/get_started_with_google_artifact_registry.ipynb

```diff
@@ -539,7 +440,7 @@
       "source": [
         "### Deleting your private Docker repostory\n",
         "\n",
-        "Finally, once your private repository becomes obsolete, use the command `gcloud artifacts repositories delete` to delete it `Google Artifact Registry`."
+        "Finally, once your private repository becomes obsolete, use the `gcloud artifacts repositories delete` command to delete it Google Artifact Registry."
```

Comment:
seems "remove....from the" is missing:

- "Finally, once your private repository becomes obsolete, use the `gcloud artifacts repositories delete` command to remove the repository from the Google Artifact Registry."